internal/hooks/auth_hooks.go

```diff
@@ -108,6 +108,7 @@ type AccessTokenClaims struct {
 	AuthenticatorAssuranceLevel   string                 `json:"aal,omitempty"`
 	AuthenticationMethodReference []models.AMREntry      `json:"amr,omitempty"`
 	SessionId                     string                 `json:"session_id,omitempty"`
+	HasVerifiedFactor             bool                   `json:"has_verified_factor"`
```

Comment:
Should be set to false if there are no factors / nil slice so there's no need for an omitempty specifier